pkg/api/types.go

```diff
@@ -3755,6 +3755,10 @@ type ResourceQuotaSpec struct {
 	// If not specified, the quota matches all objects.
 	// +optional
 	Scopes []ResourceQuotaScope
+	// If specified, indicates this ResourceQuota applies to to that priority class
+	// If not specified, this ResourceQuota applies to all priority classes.
+	// +optional
+	PriorityClassName string
```

Comment:
I am thinking that we probably need to add an *int32 for priority value as well, similar to what we did for pod. Otherwise, the class name must be resolve every time it is being used.